README.md

```diff
@@ -67,36 +67,34 @@ ide.json.path=subfolder/from/root
 file.json.name=newFileName.json
 ```
 
-**Step 1:** Install `RegisterCompanion` onto the device where you want to mock In-app Billing. You can find the latest version in the [Releases Tab](https://github.com/nytm/Register/releases).
-
-**Step 2:** Add Register as a dependency to your client app:
+**Step 1:** Add Register as a dependency to your client app, this will install the companion app:
```

Comment:
I think we should keep the point about where to find the latest releases. Also, to install the Register companion app the dependency is `register-app`